build.sbt

```diff
@@ -40,6 +40,9 @@ val standardSettings = Seq[Setting[_]](
     "Guardian GitHub Releases" at "https://guardian.github.com/maven/repo-releases",
     "Guardian GitHub Snapshots" at "https://guardian.github.com/maven/repo-snapshots"
   ),
+  riffRaffManifestProjectName := s"mobile-n10n:${name.value}",
+  riffRaffUploadArtifactBucket := Option("riffraff-artifact"),
+  riffRaffUploadManifestBucket := Option("riffraff-builds"),
```

Comment:
If you're on the latest [version](https://index.scala-lang.org/guardian/sbt-riffraff-artifact/artifacts/sbt-riffraff-artifact) of `sbt-riffraff-artifact`, you can drop these. Since https://github.com/guardian/sbt-riffraff-artifact/pull/63, they have sensible defaults 🎉 .

Reply:
Thanks :) If I've understood correctly [this](https://github.com/guardian/mobile-n10n/pull/661) PR implements your suggestions